dubbo-common/src/main/java/org/apache/dubbo/common/extension/ExtensionLoader.java

```diff
@@ -576,14 +576,10 @@ private IllegalStateException findException(String name) {
             }
         }
         StringBuilder buf = new StringBuilder("No such extension " + type.getName() + " by name " + name);
-
+        buf.append(", possible causes: ");
 
         int i = 1;
         for (Map.Entry<String, IllegalStateException> entry : exceptions.entrySet()) {

```

Comment:
if exceptions is empty, we should not print `, possible causes: `

Reply:
Yes, let's add a check when exceptions. EntrySet ().size() is greater than 0

Reply:
but my idea is we already have `i` in here, so we don't need calculate exceptions size again.
it's more like a trade off, so I would like to keep original code.

Reply:
Ok, I see

Reply:
Do I close this PR now

Reply:
apologize again for the mistake when the first time I check the issue your raised.
I'm going to close this PR. thanks again